CommandLine/libEGM/CMakeLists.txt

```diff
@@ -11,7 +11,15 @@ project(${LIB} VERSION ${LIB_VERSION} DESCRIPTION ${LIB_DESCRIPTION})
 file(GLOB SRC_FILES *.cpp)
 get_filename_component(ENIGMA_DIR "${CMAKE_CURRENT_SOURCE_DIR}" PATH)
 get_filename_component(ENIGMA_DIR "${ENIGMA_DIR}" PATH)
-add_library(${LIB} STATIC ${SRC_FILES} "${ENIGMA_DIR}/shared/lodepng/lodepng.cpp")
+
+if(MSVC)
```

Comment:
Rather than checking MSVC, maybe we ought to just add our own static define or something. Because technically you can static build on Linux and you can dynamic build on Windows. In my opinion, the behavior shouldn't depend on the compiler.